packages/halo/.storybook/config.js

```diff
@@ -7,10 +7,8 @@ import themeDecorator from '@pluralsight/ps-design-system-storybook-addon-theme'
 addDecorator(centerDecorator)
 addDecorator(themeDecorator(addons))
 
-const req = require.context('../stories', true, /\.story\.js$/)
-
 function loadStories() {
-  req.keys().forEach(filename => req(filename))
+  require('../stories')
```

Comment:
i learned this only can import a single file...not a dir of files. fyi

Reply:
probably ok in practice, but maybe want to rename the function .... `loadStoriesFile`?  Maybe?